docs/00-manuscripts/ieee/reviews/reviewer4.md

```diff
@@ -89,22 +93,21 @@ Additionally it would mainly cause more guessing and move focus away from the ma
 We think that there might not be a perfect solution to this and all proposed ones (including doing nothing at all) will have some downside.
 With respect to the suggestion to use segmentation: we agree that this would be an interesting option.
 However note that there is no problem identifying the trees or their exact location.
-The issue is with the potential offset of the hyperspectral image of up to 1m, possibly resulting in a non-match of tree and its actual corresponding pixel.
+The issue is with the potential offset of the hyperspectral image of up to 1 m, possibly resulting in a non-match of a tree point and its actual corresponding pixel.
 
 > - As the equation of the NRIs , the generic term of the normalized indices is used (NDVI). However, there is a variety of other structured types of indices – please justify your choice.
 
 We would like to thank the reviewer for pointing out the existence of alternative base formulas for NRI calculation.
 In general the NRI concept is based on the "Optimized multiple narrow-band reflectance" (OMNBR) approach by [Thenkabail et al. (2000)](thenkabail2000) and also described in [Thenkabail et al 2018](https://www.taylorfrancis.com/books/hyperspectral-indices-image-classifications-agriculture-vegetation-prasad-thenkabail-john-lyon-alfredo-huete/e/10.1201/9781315159331).
-One reasons to use the NDVI base as the index for NRI calculation was the availability in the R package [{hsdar}](https://cran.r-project.org/web/packages/hsdar/index.html) in the function [nri()](https://rdrr.io/cran/hsdar/man/normalized.ratio.index.html) which uses the NDVI-index as the base.
-The use of NRI indices was thought as an addition to the existing feature sets (HR and VI) and using only a generic NDVI formula as the base was sufficient in our view.
+One reason to use the NDVI-type index formulation for NRI calculation was its availability in the R package [{hsdar}](https://cran.r-project.org/web/packages/hsdar/index.html) in the function [nri()](https://rdrr.io/cran/hsdar/man/normalized.ratio.index.html). Other indices, e.g. obtained by simple ratioing, would be expected to be very strongly correlated with the corresponding NRIs. Corresponding the already remarkably high dimensionality of feature space, it would be less likely that such further enhancement of feature space would lead to any improvements. Our data and code are of course available for follow-up studies that might focus on different feature extraction strategies.
```

Comment:
```suggestion
One reason to use the NDVI-type index formulation for NRI calculation was its availability in the R package [{hsdar}](https://cran.r-project.org/web/packages/hsdar/index.html) in the function [nri()](https://rdrr.io/cran/hsdar/man/normalized.ratio.index.html).
Other indices, e.g. obtained by simple ratioing, would be expected to be very strongly correlated with the corresponding NRIs.
Corresponding the already remarkably high dimensionality of feature space, it would be less likely that such further enhancement of feature space would lead to any improvements.
Our data and code are of course available for follow-up studies that might focus on different feature extraction strategies.
```